README.md

```diff
@@ -119,6 +118,22 @@ The container exposes the following ports:
 
 Use postman to hit the health endpoint above to check the health status.
 
+## Run prometheus docker container
+
+- Download the specific image of Prometheus by running `docker pull prom/prometheus:v2.43.0`
```

Comment:
just a question really, but if we have a docker-compose file why are we instructing users to explicitly pull images? which btw is not necessary as an explicit step even if you run it with plain docker

Reply:
I followed the same steps used for Jaeger to be consistent.
I am unaware of the reason, maybe @zimny know the reason.